subgraph/src/datasources/ScannerPoolRegistry.ts

```diff
@@ -37,6 +38,11 @@ export function handleScannerPoolRegistered(event: ScannerPoolRegisteredEvent):
   scannerPool.owner = to.id;
   scannerPool.chainId = event.params.chainId.toI32();
   scannerPool.status = "Not Delegating"
+
+  // the source of truth is the RewardsDistributorContract (defaultFeeBps())
+  // this is a quick and simple way to have an expected default
+  scannerPool.commission = BigDecimal.fromString("10000") // 10000 == 100%
```

Comment:
If we know the proxy address of the rewards distributor (it is in network.json and subgraph.yaml) could we query it directly?

I would be afraid of the subgraph showing a different value than what is actually in the contract

Reply:
(We concluded in our private discussion that hardcoding is easier and reliable enough for now and we can pursue ways to make these kind of queries more dynamically later.)